core/build.gradle

```diff
@@ -12,7 +14,7 @@ ext {
     coroutines_version = '1.6.4'
```

Comment:
I think we need to update the coroutines version because we have updated the kotlin version